lib/elixir/src/elixir_import.erl

```diff
@@ -54,7 +54,12 @@ record_warn(Meta, Ref, Opts, Added, E) ->
 calculate(Meta, Key, Opts, Old, File, Existing) ->
   New = case keyfind(only, Opts) of
     {only, Only} when is_list(Only) ->
-      ensure_keyword_list(Meta, File, Only, only),
+      ok = ensure_keyword_list(Meta, File, Only, only),
+      case keyfind(except, Opts) of
+        false -> ok;
+        _ ->
+          elixir_errors:compile_error(Meta, File, ":only and :except cannot be supplied together")
```

Comment:
should we say ":only and :except can only be supplied together when :only is :functions or :macros"?